services/spar/src/Spar/Scim.hs

```diff
@@ -115,32 +116,33 @@ apiScim =
     -- Let's hope that SCIM clients can handle non-SCIM-formatted errors
     -- properly. See <https://github.com/haskell-servant/servant/issues/1022>
     -- for why it's hard to catch impure exceptions.
-    wrapScimErrors :: Spar a -> Spar a
-    wrapScimErrors = over _Spar $ \act -> \env -> do
-      result :: Either SomeException (Either SparError a) <- try (act env)
-      case result of
-        Left someException -> do
-          -- We caught an exception that's not a Spar exception at all. It is wrapped into
-          -- Scim.serverError.
-          pure . Left . SAML.CustomError . SparScimError $
-            Scim.serverError (cs (displayException someException))
-        Right err@(Left (SAML.CustomError (SparScimError _))) ->
-          -- We caught a 'SparScimError' exception. It is left as-is.
-          pure err
-        Right (Left sparError) -> do
-          -- We caught some other Spar exception. It is rendered and wrapped into a scim error
-          -- with the same status and message, and no scim error type.
-          err :: ServerError <- sparToServerErrorWithLogging (sparCtxLogger env) sparError
-          pure . Left . SAML.CustomError . SparScimError $
-            Scim.ScimError
-              { schemas = [Scim.Schema.Error20],
-                status = Scim.Status $ errHTTPCode err,
-                scimType = Nothing,
-                detail = Just . cs $ errBody err
-              }
-        Right (Right x) -> do
-          -- No exceptions! Good.
-          pure $ Right x
+    wrapScimErrors :: Spar r a -> Spar r a
+    wrapScimErrors act = Spar $
```

Comment:
This change looks bigger than it is --- it's mostly whitespace. It's a result of removing the `_Spar` iso, which is too impredicative to define with the packed `Member (Final IO)` dict.